samples/snippets/csharp/VS_Snippets_Misc/cancellation/cs/cancellationex11.cs

```diff
@@ -40,24 +40,25 @@ static void Main()
    //<snippet3>
    static void NestedLoops(Rectangle rect, CancellationToken token)
    {
-      for (int x = 0; x < rect.columns && !token.IsCancellationRequested; x++) {
-         for (int y = 0; y < rect.rows; y++) {
+      int col = 0;
+      for (; col < rect.columns && !token.IsCancellationRequested; col++) {
```

Comment:
I'm not a fan of having the `col` variable being declared outside of the `for` loop simply for the sake of logging it. I'd argue that we should delete line 57 altogether and instead declare this inline.
```suggestion
      for (int col = 0; col < rect.columns && !token.IsCancellationRequested; col++) {
```